packages/charts/src/chart_types/xy_chart/rendering/points.ts

```diff
@@ -105,7 +111,7 @@ export function renderPoints(
         },
         seriesIdentifier,
         panel,
-        orphan,
+        isolated: isPointIsolated,
```

Comment:
Do we care that this depends on `isolatedPointThemeStyle.visible` being `true`/`false`? If this is accessible to the consumer I think we change this to...

```suggestion
        isolated: isIsolatedPoint(dataIndex, dataSeries.data.length, yDefined, prev, next),
```

Reply:
fixed through  a435a01 and 20868e0